spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :request do # rubocop:disable Metrics/BlockLength
+  headers = { 'Authorization': "Token token=#{ENV['HOWITZER_TOKEN']}" }
+
+  describe 'GET #index' do
+    let!(:articles) { create_list(:article, 10) }
+    before { get api_v1_articles_path, headers: headers }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'returns the list of articles' do
+      expect(json.map { |x| x['id'] }).to eq [*1..10]
```

Comment:
It is bad approach too. Technically ids can start from another number. I recommend to test count as a simplest solution. See article http://aalvarez.me/blog/posts/testing-a-rails-api-with-rspec.html

Reply:
done